ppafm/common.py

```diff
@@ -57,11 +58,174 @@
     'tip_base':  np.array( ['None', 0.00 ]),
     'Rtip'         :  30.0,
     'permit'       :  0.00552634959,
-    'Vrange':   0.0,
     'vdWDampKind' : 2,
     '#' : None
 }
 
+class CLIParser(ArgumentParser):
```

Comment:
Aha it is extension of ArgumentParser. Is it some common practice to make you own derived class of ArgumentParser?

Reply:
It's maybe not common practice, but I think a good compromise. The main concern for me was that the arguments were not consistent with the names in the `params` dictionary. I wrote about the example of the amplitude being read wrong in one of the scripts in the associated issue #148. If we add the arguments through this interface, it's less error prone, there is less repeated code, and the help messages are more consistent.